src/etc/htmldocck.py

```diff
@@ -135,6 +136,12 @@
     unichr = chr
 
 
+# Handle running from a directory other than the top level
+rev_parse = ["git", "rev-parse", "--show-toplevel"]
+top_level = subprocess.check_output(rev_parse, universal_newlines=True).strip()
+channel_line = next(x for x in open(top_level + "/src/ci/run.sh").readlines() if x.startswith("export RUST_RELEASE_CHANNEL="))
```

Comment:
This is probably wrong - if there's a way to get x.py to pass the `channel` in config.toml to htmldocck instead that seems like a better approach.

Reply:
Done.